### DISCUSSION TITLE
Edu  Login Help

### DISCUSSION THREAD

**Discussion by rinrinx2** [spencerwooo/onedrive-vercel-index#272] [Q&A]
OneDrive edu accounts not working.  It's asking for admin permission.  so this index is not very functional for OneDrive.

**Comment by spencerwooo** [comment]
you can generate your own client id and secret and auth with those tokens if you need admin privileges

**Reply by spencerwooo** [reply]
i will try to get this part of the doc written in a few days (located here: https://docs.spencerwoo.com/using-your-own-client-id-and-secret)

**Reply by rinrinx2** [reply]
Unfortunately I do not have admin authority so it is very difficult to use OneDrive Business accounts.

**Reply by rinrinx2** [reply]
It will be great if you can prepare a detailed doc about how to add accounts such as private domain or edu.

**Reply by rinrinx2** [reply]
Bro its work edu thanks 🍅🍅🍅
@spencerwooo

**Reply by hirockstar** [reply]
@kamileecher tell us how and make video as well you did befor if you can.

**Reply by rinrinx2** [reply]
@therockstarind ok bro i will make a detailed video

**Reply by rinrinx2** [reply]
@therockstarind I have 10 edu accounts, I managed to connect all of them to index
![Screenshot_20220115-103418_Chrome](https://user-images.githubusercontent.com/58888200/149613777-a9e82638-1a32-4f60-a20c-2dca0e4a1ed4.jpg)

**Reply by hirockstar** [reply]
@kamileecher all account at one place wow thats amazing please tell us too how i want to add too.

**Comment by rinrinx2** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/58888200/149357983-19ae717b-a9c9-4b3c-bf94-a91bf9b66f01.png)


@spencerwooo what problem

**Reply by spencerwooo** [reply]
https://docs.spencerwoo.com/getting-started#53e8f64f3b18466dab3b695a49bde346

**Reply by spencerwooo** [reply]
> Here we will validate that whether you are actually you, so if the button says “Don’t pretend to be the site owner”, go change the userPrincipalName under config/site.json, which is definitely wrong.
